lib/services/livesync/livesync-service.ts

```diff
@@ -109,6 +109,9 @@ class LiveSyncService implements ILiveSyncService {
 				this.$hooksService.executeBeforeHooks('watch').wait();
 				this.partialSync(liveSyncData[0].syncWorkingDirectory, watchForChangeActions);
 			}
+			if (this.$options.justlaunch) {
+				process.kill(process.pid);
```

Comment:
No, this isn't process.exit. Process.exit exits the app unconditionally. This one simulates Ctrl+C and all ongoing tasks registered on SIGKILL will execute.